packages/mds-collector-backend/@types/index.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2021 City of Los Angeles
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { RpcRoute, RpcServiceDefinition } from '@mds-core/mds-rpc-common'
+import { DomainModelCreate } from '@mds-core/mds-repository'
+import { NonEmptyArray, Timestamp, UUID } from '@mds-core/mds-types'
+
+export interface CollectorMessageDomainModel {
+  schema_id: string
+  producer_id: UUID
+  message: {}
```

Comment:
I think you probably want `object` as the type here, not `{}`. More restrictive

Reply:
The idea is to allow anything that can be described by a schema, but I can change to `object` if we don't think we want to ingest strings or numbers alone.